Tasks/DotNetCoreCLIV2/task.json

```diff
@@ -557,6 +557,7 @@
         "NGCommon_UnabletoDetectNuGetVersion": "Unknown NuGet version selected.",
         "NGCommon_UnableToFindTool": "Unable to find tool %s",
         "Warning_SessionCreationFailed": "Could not create provenance session: %s",
-        "Warning_UpdatingNuGetVersion": "Updating version of NuGet.exe to %s from %s. Behavior changes or breaking changes might occur as NuGet updates to a new version. If this is not desired, deselect the 'Check for Latest Version' option in the task."
+        "Warning_UpdatingNuGetVersion": "Updating version of NuGet.exe to %s from %s. Behavior changes or breaking changes might occur as NuGet updates to a new version. If this is not desired, deselect the 'Check for Latest Version' option in the task.",
+        "NetCore3Update": "Info: Our Hosted Agents have been updated to contain .Net Core 3.0 SDK/Runtime along with 2.1 and 2.2. with .NET Core CLI 3.x version. Due to 3.0 version being present in the system, you might see difference in behaviour of .NET Core. \nSome commonly encountered behavoiur changes due to .NET Core CLI 3.0, are: \nIf you're using `Publish` command with -o or --Output argument, you will see that the output folder is not being created at root directory rather than Project File's directory.To learn about more such changes and troubleshoot, refer here: https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/build/dotnet-core-cli?view=azure-devops#troubleshooting"
```

Comment:
```suggestion
        "NetCore3Update": "Info: Azure Pipelines Hosted Agents have been updated to contain .Net Core 3.x SDK/Runtime along with 2.x. Unless you have locked down a SDK version for you project(s), 3.x SDK might be picked up which might have breaking behavior as compared to previous versions. \nSome commonly encountered changes are: \nIf you're using `Publish` command with -o or --Output argument, you will see that the output folder is now being created at root directory rather than Project File's directory. To learn about more such changes and troubleshoot, refer here: https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/build/dotnet-core-cli?view=azure-devops#troubleshooting"
```